### PR TITLE
[Release crash fix] Correct processing fake feature of IndexGraphStarter.

### PR DESCRIPTION
**Pull request by bykoianko**
@ygorshenin @dobriy-eeh PTAL

### CODE REVIEW

File: routing/index_graph_starter.cpp [hunk 1]
```diff
@@ -40,20 +40,34 @@ m2::PointD const & IndexGraphStarter::GetPoint(Joint::Id jointId)
...
-    if (e.GetFeatureId() == rp.GetFeatureId())
-      return GetPoint(e.GetFrom());
+    if (e.GetFeatureId() == fakeFeatureId)
+      return e;
   }
-  CHECK(false,
-        (rp, "is a point of a fake feature but it's not contained in |m_fakeZeroLengthEdges|"));
-  return m_graph.GetPoint(rp);
+  CHECK(false, ("Fake feature:", fakeFeatureId, "has to be contained in |m_fakeZeroLengthEdges|"));
+  return m_fakeZeroLengthEdges.front();
```
Comment (ygorshenin) [inline routing/index_graph_starter.cpp]: Better to return default-constructed DirectedEdge, if it's possible.

Comment (bykoianko) [inline routing/index_graph_starter.cpp]: Yes. But in this case there's a warning about returning local value by const ref if to do so.

Comment (bykoianko) [inline routing/index_graph_starter.cpp]: No problem in returning DirectedEdge by value except for little number of copying a small object (DirectedEdge) in seldom cases. On the other hand it's little cost we have to pay for returning default-constructed DirectedEdge. As for me the both variants are ok. Do you prefer returning a default - constructed DirectedEdge by value?

Comment (ygorshenin) [inline routing/index_graph_starter.cpp]: It's up to you.

LGTM in any case.

File: routing/index_graph_starter.cpp [hunk 2]
```diff
@@ -40,20 +40,34 @@ m2::PointD const & IndexGraphStarter::GetPoint(Joint::Id jointId)
...
+{
+  if (!IsFakeFeature(rp.GetFeatureId()))
+    return m_graph.GetPoint(rp);
+
+  // Fake edges have zero length so coords of "point from" and "point to" are equal.
+  return GetPoint(FindFakeEdge(rp.GetFeatureId()).GetFrom());
+}
+
+RoadGeometry IndexGraphStarter::GetFakeRoadGeometry(uint32_t fakeFeatureId)
+{
```
Comment (dobriy-eeh) [inline routing/index_graph_starter.cpp]: ASSERT( IsFakeFeature(fakeFeatureId), ("Feature" + fakeFeatureId "is not fake" ));
?

Comment (bykoianko) [inline routing/index_graph_starter.cpp]: FindFakeEdge вызывается от сюда.

**Comment by dobriy-eeh** [comment]
LGTM

**Pull request merged**